diff --git a/ppp-2.4.7/debian/rules b/ppp-2.4.7/debian/rules
index 972fa87..c1ae846 100755
--- a/ppp-2.4.7/debian/rules
+++ b/ppp-2.4.7/debian/rules
@@ -30,6 +30,10 @@ UDEB_MAKEOPTS := \
 	USE_BUILTIN_CRYPTO=1 USE_EAPTLS=
 UDEB_PLUGIN_DIRS := pppoatm rp-pppoe
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+	CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 ##############################################################################
 %:
 	dh $@
@@ -79,14 +83,14 @@ changelog-from-README:
 
 override_dh_auto_build-arch: changelog-from-README
 	dh_auto_build -- \
-		COPTS="$(CFLAGS) $(CPPFLAGS)" LDOPTS="$(LDFLAGS)"
+		COPTS="$(CFLAGS) $(CPPFLAGS)" LDOPTS="$(LDFLAGS)" CC="$(CC)"
 	dh_auto_build -Dpppd-udeb/ -- \
 		COPTS="$(CFLAGS) $(CPPFLAGS) -Os -fomit-frame-pointer" \
 		LDOPTS="$(LDFLAGS)" \
-		$(UDEB_MAKEOPTS)
+		$(UDEB_MAKEOPTS) CC="$(CC)"
 	dh_auto_build -Dpppd-udeb/plugins -- \
 		COPTS="$(CFLAGS) $(CPPFLAGS) -Os -fomit-frame-pointer" \
-		LDOPTS="$(LDFLAGS)" \
+		LDOPTS="$(LDFLAGS)" CC="$(CC)" \
 		SUBDIRS="$(UDEB_PLUGIN_DIRS)" PLUGINS=
 
 override_dh_auto_build-indep: changelog-from-README
