diff --git a/pacman-10/debian/rules b/pacman-10/debian/rules
index d4b79b0..37656ea 100755
--- a/pacman-10/debian/rules
+++ b/pacman-10/debian/rules
@@ -15,13 +15,21 @@ else
 endif
 export CFLAGS
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+	CXX=$(DEB_HOST_GNU_TYPE)-g++
+	CCLINK=$(DEB_HOST_GNU_TYPE)-g++
+else
+	CXX=g++
+	CCLINK=g++
+endif
+
 build: build-arch build-indep
 build-arch: build-stamp
 build-indep: build-stamp
 build-stamp:
 	dh_testdir
 	xmkmf -a
-	$(MAKE)
+	$(MAKE) CXX=$(CXX) CCLINK=$(CCLINK)
 	touch build-stamp
 
 clean:
