diff --git a/mdadm-4.1~rc1/debian/rules b/mdadm-4.1~rc1/debian/rules
index b13f218..5072946 100755
--- a/mdadm-4.1~rc1/debian/rules
+++ b/mdadm-4.1~rc1/debian/rules
@@ -7,6 +7,8 @@
 #export DH_VERBOSE=1
 
 export CROSS_COMPILE=$(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)-
+export CC = $(CROSS_COMPILE)gcc
+
 LDFLAGS = $(shell dpkg-buildflags --get LDFLAGS)
 CXFLAGS = $(shell dpkg-buildflags --get CFLAGS) \
 	  $(shell dpkg-buildflags --get CPPFLAGS)
