diff --git a/buffer-1.19/debian/rules b/buffer-1.19/debian/rules
index 604315e..5aac96a 100755
--- a/buffer-1.19/debian/rules
+++ b/buffer-1.19/debian/rules
@@ -14,9 +14,13 @@ build-arch: build-stamp
 .PHONY: build-indep
 build-indep: build-stamp
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+        CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 build-stamp:
 	dh_testdir
-	make CFLAGS="$(CFLAGS) $(CPPFLAGS)" LDFLAGS="$(LDFLAGS)"
+	make CC="$(CC)" CFLAGS="$(CFLAGS) $(CPPFLAGS)" LDFLAGS="$(LDFLAGS)"
 ifeq (, $(filter nocheck,$(DEB_BUILD_OPTIONS)))
 	make check
 endif
