diff --git a/blktrace-1.1.0/debian/rules b/blktrace-1.1.0/debian/rules
index 754ab3e..e620852 100755
--- a/blktrace-1.1.0/debian/rules
+++ b/blktrace-1.1.0/debian/rules
@@ -12,6 +12,10 @@ UPSTREAMVER:=$(shell echo $(FULLVERSION) | sed 's/-[0-9.]\+$$//' | sed 's/^0~//'
 
 export DEB_BUILD_MAINT_OPTIONS=hardening=+all
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+	CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 %:
 	dh $@
 
@@ -32,7 +36,7 @@ else
 	####################################################
 	false
 endif
-	$(MAKE) CFLAGS="$(CPPFLAGS) $(CFLAGS) $(LDFLAGS)" all docs
+	$(MAKE) CC="$(CC)" CFLAGS="$(CPPFLAGS) $(CFLAGS) $(LDFLAGS)" all docs
 
 override_dh_auto_install:
 	$(MAKE) install \
