Description: Add missing configuration option

diff --git a/gcal-3.6.3/debian/rules b/gcal-3.6.3/debian/rules
index eda497b..8ed7d7c 100755
--- a/gcal-3.6.3/debian/rules
+++ b/gcal-3.6.3/debian/rules
@@ -23,7 +23,8 @@ endif
 override_dh_auto_build:
 	PAGER1=pager ./configure --prefix=/usr \
 		--mandir=\$${prefix}/share/man \
-		--infodir=\$${prefix}/share/info 
+		--infodir=\$${prefix}/share/info \
+		--host=$(DEB_HOST_GNU_TYPE)
 	$(MAKE) CFLAGS="$(CFLAGS) $(CPPFLAGS)" LDFLAGS="$(LDFLAGS)"
 
 override_dh_clean:
