diff --git a/ifstat-1.1/debian/rules b/ifstat-1.1/debian/rules
index c1a1233..94aaa3d 100755
--- a/ifstat-1.1/debian/rules
+++ b/ifstat-1.1/debian/rules
@@ -9,6 +9,10 @@
 # This handles nostrip (via dh_strip) but not noopt. The configure.in
 # overrides CFLAGS and needs to be edited to disable optimisation.
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+export CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 configure: configure-stamp
 configure-stamp:
 	dh_testdir
