--- agg-2.5+dfsg1/debian/rules	2017-06-14 14:32:12.229427629 +0700
+++ agg-2.5+dfsg1/debian/rules	2017-06-14 14:42:04.981421329 +0700
@@ -2,7 +2,7 @@
 
 FLAVOURS := pic nonpic
 
-CONFIGURE_ARGS_COMMON := --prefix=/usr --disable-examples --x-includes=/usr/include --x-libraries=/usr/lib
+CONFIGURE_ARGS_COMMON := --prefix=/usr --disable-examples --x-includes=/usr/include --x-libraries=/usr/lib --build=$(DEB_BUILD_GNU_TYPE) --host=$(DEB_HOST_GNU_TYPE)
 CONFIGURE_ARGS_pic := -with-pic
 CONFIGURE_ARGS_nonpic := --disable-gpc
 DESTDIR_nonpic := debian/libagg-dev
