--- libxvmc-1.0.10/debian/rules
+++ libxvmc-1.0.10/debian/rules
@@ -1,7 +1,19 @@
 #!/usr/bin/make -f
 
+# cannot run test code for malloc0returnsnull when cross compile
+ifneq ($(DEB_HOST_GNU_TYPE),$(DEB_BUILD_GNU_TYPE))
+	ifeq (,$(findstring malloc0returnsnull,$(DEB_BUILD_OPTIONS)))
+		confflags += --disable-malloc0returnsnull
+	else
+		confflags += --enable-malloc0returnsnull
+	endif
+endif
+
 %:
 	dh $@
 
+override_dh_auto_configure:
+	dh_auto_configure -- $(confflags)
+
 override_dh_install:
 	dh_install --fail-missing --exclude=.la
