diff --git a/csh-20110502/debian/rules b/csh-20110502/debian/rules
index 426c8d4..2e19136 100755
--- a/csh-20110502/debian/rules
+++ b/csh-20110502/debian/rules
@@ -3,6 +3,10 @@
 # Standard options minus `-Wall', since upstream uses `-Werror'.
 export CFLAGS := -g -O$(if $(findstring noopt,$(DEB_BUILD_OPTIONS)),0,2) -Wno-error=unused-result
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+export	CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 %:
 	dh $@ --buildsystem=bmake
 
