diff --git a/tinycdb-0.78/debian/rules b/tinycdb-0.78/debian/rules
index bc99353..d842b50 100755
--- a/tinycdb-0.78/debian/rules
+++ b/tinycdb-0.78/debian/rules
@@ -1,6 +1,10 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+	CC = $(DEB_HOST_GNU_TYPE)-gcc
+endif
+
 LIBDIR = /usr/lib/$(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 CFLAGS = $(shell dpkg-buildflags --get CFLAGS) \
@@ -16,7 +20,7 @@ configure:	# nothing
 build-arch: libcdb.pc
 libcdb.pc:
 	dh_testdir
-	$(MAKE) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" \
+	$(MAKE) CC="$(CC)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" \
 		staticlib sharedlib cdb-shared nss
 	cp -pf nss_cdb-Makefile cdb-Makefile
 	cp -pf cdb-shared cdb
