diff --git a/epcr-2.3.12-1/debian/rules b/epcr-2.3.12-1/debian/rules
index 8ed4bd5..37577c6 100755
--- a/epcr-2.3.12-1/debian/rules
+++ b/epcr-2.3.12-1/debian/rules
@@ -2,6 +2,10 @@
 # debian/rules for ncbi-epcr using
 # Andreas Tille <tille@debian.org>, GPL
 
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+export CXX = $(DEB_HOST_GNU_TYPE)-g++
+endif
+
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 VERSION := $(shell dpkg-parsechangelog | sed -ne 's/^Version: \(\([0-9]\+\):\)\?\(.*\)-.*/\3/p')
